site/content/blog/introducing-ckan-mcp-server-connect-ai-assistants-to-your-data-portals.md

```diff
@@ -0,0 +1,219 @@
+
+![MCP to LLM Connection](/static/img/blog/introducing-ckan-mcp-server-connect-ai-assistants-to-your-data-portals/mcp-llm.png)
+
+Before diving into our CKAN implementation, let's understand what makes MCP so revolutionary. Announced by Anthropic in November 2024, the Model Context Protocol is quickly becoming the universal standard for connecting AI assistants to data systems.
+
+### The Problem MCP Solves
+
+Traditional AI assistants are "trapped behind information silos and legacy systems." Every new data source requires custom implementation, making truly connected systems difficult to scale. Until now, connecting an AI assistant to your organization's data meant:
+
+- Building custom integrations for each data source
+- Managing fragmented, bespoke connections
+- Dealing with security and access control complexities
+- Maintaining multiple different protocols and standards
+
+### The MCP Solution
+
+MCP replaces these fragmented integrations with a single, open protocol. Think of it as the "USB standard" for AI-data connections - any MCP-compliant data source can serve context to any MCP-enabled AI client, and vice versa.
+
+## Industry Adoption: The New Standard for AI
+
+The adoption of MCP has been remarkable. Major tech companies are rapidly integrating it:
+
+- **OpenAI** officially adopted MCP in March 2025, integrating it across ChatGPT, their Agents SDK, and Responses API
+- **Google DeepMind** announced MCP support in Gemini models, with CEO Demis Hassabis calling it "rapidly becoming an open standard for the AI agentic era"
+- **Microsoft** unveiled plans to make MCP a "foundational layer for secure, interoperable agentic computing" in Windows 11
+
+By early 2025, the community had already created over 1,000 MCP servers, covering everything from GitHub and Slack to databases and enterprise systems.
+
+## Introducing the CKAN MCP Server
+
+### What is CKAN?
+
+CKAN (Comprehensive Knowledge Archive Network) is the world's leading open-source data management system, used by governments, research institutions, and organizations worldwide to publish, share, and manage datasets. From national open data portals to research repositories, CKAN powers some of the most important data infrastructure on the web.
+
+### Why Connect CKAN to AI Assistants?
+
+Imagine being able to:
+
+- **Ask natural language questions** about datasets: "Show me environmental datasets from 2020"
+- **Get instant summaries** of complex data without browsing through catalogs
+- **Discover relationships** between datasets across different organizations
+- **Access metadata** and resources through conversational interfaces
+- **Integrate data discovery** into your existing AI workflows
+
+This is exactly what the CKAN MCP Server enables.
+
+## How It Works: Technical Overview
+
+### Architecture
+
+![MCP Server Architecture Diagram](/static/img/blog/introducing-ckan-mcp-server-connect-ai-assistants-to-your-data-portals/mcp-server-diagram.png)
```

Comment:
I must say this diagram is confusing to me, i don't think it is accurate. I'd remove it.